docs/source/configuration.md

```diff
@@ -20,6 +20,10 @@ Following fields are required (and can be customized) in the [`challenge_config.
 
 - **remote_evaluation**: True/False (specify whether evaluation will happen on a remote machine or not. Default is `False`)
 
+- **is_docker_based**: True/False (specify whether the challenge is docker based or not. Default is `False`)
+
+- **is_static_dataset_code_upload**: True/False (specify whether the challenge is static dataset code upload or not. Default is `False`)
```

Comment:
Can we please define what does `static dataset code upload` means?

Reply:
I don't know if we had defined it at some place in our documentation. If not, then we should define it something like it is a combination of both prediction upload and code upload challenge.

cc: @Ram81

Reply:
I have defined it for the code-upload PR: https://github.com/Cloud-CV/EvalAI/pull/3865